packages/trader/src/AppV2/Containers/Trade/__tests__/trade-types.spec.tsx

```diff
@@ -99,4 +99,15 @@ describe('TradeTypes', () => {
         unmount();
         expect(default_mock_store.modules.trade.onUnmount).toHaveBeenCalled();
     });
+
+    it('should scroll to the selected trade type when tradeList is clicked', async () => {
+        render(mockTradeTypes());
+        const scrollByMock = jest.fn();
+        Object.defineProperty(HTMLElement.prototype, 'scrollBy', {
```

Comment:
```
    const originalScrollBy = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'scrollBy');
    beforeAll(() => Object.defineProperty(HTMLElement.prototype, 'scrollBy', { value: scrollByMock }));

    afterAll(() => Object.defineProperty(HTMLElement.prototype, 'scrollBy', originalScrollBy as PropertyDescriptor));
```

Reply:
Might it be better to clean it? Please, ignore if irrelevant, I just noticed that in some test cases in our repp some people clean, some - not

Reply:
Seems, that `Chip` from Quill can accept `ref`